build.gradle

```diff
@@ -402,6 +426,8 @@ dependencies {
     implementation 'org.apache.poi:poi'
     implementation 'org.apache.poi:poi-scratchpad'
 
+    provided "org.apache.tomcat:tomcat-el-api:${minimumTomcatVersion}"
```

Comment:
... we just want to use the el-api release Tomcat exposes (fixes a minor mistake in OBGM-363)